src/marshaller/HipieDDL.js

```diff
@@ -894,11 +900,17 @@
                     updatedViz.getInputVisualizations().forEach(function (inViz, idx) {
                         if (inViz._eventValues) {
                             for (var key in inViz._eventValues) {
+                                var changed = inViz === context;
```

Comment:
This "fix" is a hack for some missing information from HIPIE ("by design" - but will be fixed in a later version).  So essentially I am guessing what input params need to be defaulted to their existing values and which values need to be marked as "changed" - this is so the "updates" array and "request" params or correct (I hope)